### ISSUE TITLE
Presentation file

### ISSUE THREAD

**Issue by donbowen** [LeDataSciFi/the-fin388-project-1-mba-ftw#2]
@fernandotamayog Do you have the slides link?

**Comment by fernandotamayog** [comment]
Yes! Here it is: https://drive.google.com/file/d/1S6QdcOn9_P_nL_kXjhKUBq-gF-ST3Urx/view?usp=sharing please let me know if you can open it.
